### PR TITLE
feature/AT-SSO-API-21_SwaggerConfig

### PR DESCRIPTION
**Pull request by Bwore**
Swagger implementation to show available methods. 
Add dependency and the class SwaggerConfig to the project.

### CODE REVIEW

**Review by rodrigo-velasco — APPROVED** [review]

File: src/main/java/com/agilethought/internship/sso/swagger/SwaggerConfig.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfig {
+	
+	@Bean
+	public Docket api() {
+		return new Docket(DocumentationType.SWAGGER_2)
+				.select()
+				.apis(RequestHandlerSelectors.basePackage("com.agilethought.internship.sso.controller"))
```
Comment (at-internship) [inline src/main/java/com/agilethought/internship/sso/swagger/SwaggerConfig.java]: Indent Code

File: pom.xml [hunk 2]
```diff
@@ -68,6 +68,18 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-actuator</artifactId>
 		</dependency>
+		
+		<dependency>
+    		<groupId>io.springfox</groupId>
```
Comment (at-internship) [inline pom.xml]: Indent Code

**Review by at-internship — CHANGES_REQUESTED** [review]

**Review by carla-paz — APPROVED** [review]
A little tip to indent correctly Luis, ctrl+shift+f  In the pom file it is not necessary to select the code, just hit ctrl+shift+f but for the rest of the code do select portions of the code, do not try to do it all at once. Hope this helps!

**Review by carla-paz — APPROVED** [review]
Looks good!

**Review by at-internship — APPROVED** [review]

**Pull request merged**